app/src/main/res/values/strings.xml

```diff
@@ -56,17 +56,17 @@
     <string name="button_cancel">Cancel</string>
     <string name="button_ok">OK</string>
 
-    <string name="text_lack_permission">Easer does NOT have all permissions it needs.</string>
+    <string name="text_lack_permission">Easer does NOT have all the permissions it needs.</string>
     <string name="button_request_permission">Grant permissions</string>
 
     <string name="text_title">Title</string>
     <string name="text_parent_node">Parent</string>
     <string name="text_parent">Parent</string>
     <string name="text_profile">Profile</string>
     <string name="text_predefined_event">Predefined Event</string>
-    <string name="text_inline_event">Inline Event</string>
+    <string name="text_inline_event">Integrated Event</string>
```

Comment:
I'm not sure about the wording here... I guess you are aware what this string is for. Integrated doesn't sound like what a lay man would understand (though I need to say this field will be removed in the future).

Reply:
Inline proves difficult to translate, from what I remember. Hope it is gone now :)